dashboard/app/controllers/gallery_activities_controller.rb

```diff
@@ -39,7 +39,7 @@ def create
       authorize! :save_to_gallery, @gallery_activity.user_level
 
       if @gallery_activity.save
-        render action: 'show', status: :created, location: @gallery_activity
+        return head :created
```

Comment:
My impression is that the route gets set [here](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/application_controller.rb#L206-L211) and called from [here](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/feedback.js#L358-L360).